internal/services/subscription/extended_locations_data_source_test.go

```diff
@@ -0,0 +1,39 @@
+package subscription_test
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-provider-azurerm/internal/acceptance"
+)
+
+type ExtendedLocationsDataSource struct{}
+
+func TestAccDataSourceExtendedLocations_basic(t *testing.T) {
```

Comment:
since this is specific to West US we should make this specific to West US for now - and add a test for West Europe to check this returns an error when there's no extended locations